aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -93,6 +103,17 @@ Tensor quantize_per_channel(
   return quantizer->quantize(self);
 }
 
+Tensor quantize_per_channel_w_qscheme(
```

Comment:
what about adding a qscheme with default argument of per_tensor_affine/per_channel_affine to quantize_per_tensor/quantize_per_channel functions?

Reply:
Since QScheme is a special type, we cannot directly assign a default value to it in `native_functions.yaml` for existing `quantize_per_tensor` function. Let me know if you're aware of a way to do this.

Reply:
is this a limitation of codegen?

Reply:
I think we have some defaults for QScheme, like https://github.com/pytorch/pytorch/blob/master/tools/codegen/api/python.py#L283, can you try using `per_tensor_affine` in `native_functions.yaml`

Reply:
I don't think the defaults for qscheme are supported in the codegen for `native_functions.yaml`. If we add QScheme argument to `quantize_per_tensor` it throws the following error https://gist.github.com/supriyar/0165a7013761ea1dfd5118d2ee7466ca